MovieAppPetProject/MovieAppPetProject/HomeScreen/NewHomeViewController.swift

```diff
@@ -40,16 +49,24 @@ class NewHomeViewController: UIViewController, UITableViewDelegate, UITableViewD
         return 1
```

Comment:
return 1?😄 That doesn't make sense to me

Reply:
this is the number of rows per section, so I have 4 sections in the tableview and 1 row in each section. each section has 1 row because it holds the collection view